scripts/util.js

```diff
@@ -0,0 +1,41 @@
+const tmp = require("tmp");
+const ts = require("typescript");
+const path = require("path");
+const fs = require("fs-extra");
+
+tmp.setGracefulCleanup(); // cleans tmp file on process exit
+exports.corvidDir = path.join(__dirname, "../");
+
+const prepareEmptyTypescriptProgram = configPath => {
+  const tmpDir = tmp.dirSync();
+
+  const tmpDirPath = tmpDir.name;
+
+  const tmpConfigPath = `${tmpDirPath}/tsconfig.json`;
+  const tmpConfigContent = `{"extends": "${path.resolve(
+    exports.corvidDir,
+    configPath
+  )}"}`;
+
+  fs.writeFileSync(tmpConfigPath, tmpConfigContent);
+  fs.writeFileSync(`${tmpDirPath}/empty.js`, "");
```

Comment:
do you ever delete this file?

Reply:
why is it needed? where is it used?

Reply:
It is the .js file for the compiler to compile, it is created inside a tmp dir that is automatically delete on process exit thanks to `tmp.setGracefulCleanup()`